src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -97,44 +116,42 @@ public DriveTrain() {
    * @param rightSpeed the speed that the motors on the right side will run at
    */
   public void tankDrive(double leftSpeed, double rightSpeed) {
-    m_drive.tankDrive(leftSpeed, rightSpeed);
+    m_leftBack.set(leftSpeed);
+    m_rightBack.set(rightSpeed);
   }
 
   /**
    * Stops the robot
    */
   public void stop() {
-    m_drive.tankDrive(0.0, 0.0);
+    tankDrive(0.0, 0.0);
   }
 
   /**
    * @param drive the forward movement of the robot
    * @param turn  the angle that the robot would turn to
    */
   public void arcadeDrive(double drive, double turn) {
-    m_drive.arcadeDrive(drive, turn);
-  }
-
-  public void curvatureDrive(double xSpeed, double zRotation, boolean isQuickTurn) {
-    m_drive.curvatureDrive(xSpeed, zRotation, isQuickTurn);
+    m_leftBack.set(drive + turn);
+    m_rightBack.set(drive - turn);
```

Comment:
Clip, then square, then add these. 

Basically duplicate the `DifferentialDrive::arcadeDrive()`

Reply:
resolved in 93bb0d3